common/src/main/java/com/wynntils/features/user/UpdatesFeature.java

```diff
@@ -38,50 +39,51 @@ public void onWorldStateChange(WorldStateEvent event) {
 
         firstJoin = false;
 
-        CompletableFuture.runAsync(() -> {
-            UpdateManager.getLatestBuild().whenCompleteAsync((version, throwable) -> {
-                if (version == null) {
-                    WynntilsMod.info("Couldn't fetch latest version, not attempting update reminder or auto-update.");
-                    return;
-                }
-
-                if (Objects.equals(version, WynntilsMod.getVersion())) {
-                    WynntilsMod.info("Mod is on latest version, not attempting update reminder or auto-update.");
-                    return;
-                }
-
-                if (updateReminder) {
-                    remindToUpdateIfExists(version);
-                }
-
-                if (autoUpdate) {
-                    if (WynntilsMod.isDevelopmentEnvironment()) {
-                        WynntilsMod.info("Tried to auto-update, but we are in development environment.");
+        CompletableFuture.runAsync(() -> UpdateManager.getLatestBuild()
```

Comment:
I'm confused. Is this a merge error?

Reply:
No. Some really weird thing happened to MC because we sent a message to a player chat from ForkJoinPool. Now, we should send it from the correct, game thread